### ISSUE TITLE
Validator True : name error

### ISSUE THREAD

**Issue by kazenojiyu** [vihuvac/recaptcha-bundle#2]
I'am trying to use the validator without entity by using the form method:

```
            ->add('recaptcha', RecaptchaType::class, 
                array(
                    'mapped'      => false,
                    'constraints' => array(
                       new IsTrueCaptcha(),
                    )
                ));
```

But I have this error :

```
Compile Error: Cannot use Vihuvac\Bundle\RecaptchaBundle\Validator\Constraints\True as True because 'True' is a special class name
```

I'm using symfony 2.8.6.

**Comment by vihuvac** [comment]
Hi @kazenojiyu,

What do you mean without entity? You got two simple ways of validating the recaptcha, basically both of them are the same, but I recommend you to use the True validator within your formType because it's easier, e.g:

```
<?php

namespace Acme\Bundle\DemoBundle\Form\Type;

...
use Vihuvac\Bundle\RecaptchaBundle\Validator\Constraints\True;

class DemoFormType extends AbstractType
{
    public function buildForm(FormBuilderInterface $builder, array $options)
    {
        $builder
            ...
            ->add("recaptcha", "vihuvac_recaptcha",
                [
                    "label"       => "Recaptcha Label",
                    "attr"        => [
                        "options" => [
                            "theme" => "light",
                            "type"  => "audio"
                        ]
                    ],
                    "mapped"      => false,
                    "constraints" => [new True()]
                ]
            )
        ;
    }
    ...
}
```

Then your twig just need:

```
{{ form_label(form.recaptcha, null, { "label_attr": { "class": "sr-only" } }) }}
{% form_theme form "VihuvacRecaptchaBundle:Form:vihuvac_recaptcha_widget.html.twig" %}
{{ form_widget(form.recaptcha) }}
```

It works in Symfony 2*. I hope it helps!

**Comment by kazenojiyu** [comment]
Hi @vihuvac,

Thank you for your reply.
I use a form without entity attached.

```
For eg (form with an entity contains link to the entity):
    public function configureOptions(OptionsResolver $resolver)
    {
        $resolver->setDefaults(array(
            'data_class' => 'AppBundle\Entity\Category'
        ));
    }
```

But using the "True" validator in my form throw the previous error :

```
Compile Error: Cannot use Vihuvac\Bundle\RecaptchaBundle\Validator\Constraints\True as True because 'True' is a special class name
```

This is my code:

```
use Symfony\Component\Form\AbstractType;
use Symfony\Component\Form\FormBuilderInterface;
use Symfony\Component\OptionsResolver\OptionsResolver;
use Symfony\Component\Form\Extension\Core\Type\TextType;
use Symfony\Component\Form\Extension\Core\Type\TextareaType;
use Symfony\Component\Form\Extension\Core\Type\EmailType;
use Vihuvac\Bundle\RecaptchaBundle\Form\Type\RecaptchaType;
use Symfony\Component\Validator\Constraints\Email;
use Symfony\Component\Validator\Constraints\Length;
use Symfony\Component\Validator\Constraints\NotBlank;
use Vihuvac\Bundle\RecaptchaBundle\Validator\Constraints\True;

class ContactType extends AbstractType
{
    public function buildForm(FormBuilderInterface $builder, array $options)
    {
        $builder
            ->add('recaptcha', RecaptchaType::class, 
                array(
                    'mapped'      => false,
                    'constraints' => array(
                       new True(),
                    )
                ));
    }   
}
```

**Comment by vihuvac** [comment]
@kazenojiyu, yeah, the problem is that you are defining again the recaptcha type, there's not need to add `RecaptchaType::class` for the recaptcha field, just use `'vihuvac_recaptcha'` instead, the value is already defined in the bundle itself. So your recaptcha field must look like:

```
...
class ContactType extends AbstractType
{
    public function buildForm(FormBuilderInterface $builder, array $options)
    {
        $builder
            ->add('recaptcha', 'vihuvac_recaptcha', 
                array(
                    'mapped'      => false,
                    'constraints' => array(
                       new True(),
                    )
                ));
    }   
}
```

**Comment by kazenojiyu** [comment]
@vihuvac I did your modification but it didn't work.
The code `->add('recaptcha', RecaptchaType::class,` is the new call for the form type since symfony 2.8.

If I change the name of the validator, something like `IsValid`, error disappear.
I think this error is only on newest symfony version like 2.8.

Can I propose a modification of the name of the Validator ?

**Comment by vihuvac** [comment]
@kazenojiyu I think that issue is due to PHP7, you can take a look at this issue:

https://github.com/symfony/symfony/issues/14336

The same is related to other two issues. I couldn't update the bundle due to I’ve been caught up in moving a project of mine from vagrant to docker, I am going to be updating this bundle asap. Meanwhile I suggest keep on working under PHP5 or PHP6 for your Symfony Project 2.*, I got a project running with the same symfony version than your project and the recaptcha bundle works perfectly with PHP5.5.34.

The fixes are really simple to do, I just apologize for the delay. If you got a PR, you're welcome!

**Comment by vihuvac** [comment]
Issue fixed with the new release **v2.1.0**.
